datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/manifest.py

```diff
@@ -83,16 +82,6 @@ def manifest(ctx, check, fix):
                     for message in messages:
                         display_queue.append((message_methods[msg_type], message))
 
-            # Check is_public only for changed checks or specific check for reduced verbosity
```

Comment:
This isn't currently passed into the underlying validators to know if we're running against a single or all checks. So this approach means this validation will run for each integration.

If this is a hard requirement, the approach can be changed to leave the `is_public` at the root of this function, but having split manifest version logic here felt not right 🙂